streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -547,6 +584,13 @@ private void cleanGlobal(final boolean withIntermediateTopics,
         cleanUpConfig.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "" + CLEANUP_CONSUMER_TIMEOUT);
 
         final int exitCode = new StreamsResetter().run(parameters, cleanUpConfig);
+        return exitCode;
```

Comment:
Like said earlier, I think we could just return
`return new StreamsResetter().run(parameters, cleanUpConfig) == 0`

Reply:
Updated